app/api/srch/api.rb

```diff
@@ -19,9 +18,7 @@ class API < Grape::API
                               models: [
                                 SearchRequest::Entity,
                                 DocResult::Entity,
-                                TagResult::Entity,
-                                DocList::Entity,
-                                TagList::Entity
```

Comment:
Just to confirm, does the Search Service return tags? I think I remember that it may, just as part of a DocList or something.... but note how tags are returned in the site header's autocompletion search: 

![image](https://user-images.githubusercontent.com/24359/44618884-83fa4d00-a84c-11e8-8683-f99aa3fc29f1.png)

Reply:
Otherwise this cleanup looks great!

Reply:
hey, @jywarren I pushed the changes to the unstable, but I don't know why the server isn't running... anyway, about that, I spent the night fixing this. DocResult also has those attributes, so I changed the typeahead script to use the items from DocResult instead of TagResult. I can't reproduce locally nor on the unstable to send to you a print, but the last commit shows how the icons are generated. 

Interesting because now I finally get what are those DocResult params for! xD thanks for the help, I believe now this is good to go!